dist/qs.js

```diff
@@ -578,7 +578,7 @@ var merge = function merge(target, source, options) {
     if (Array.isArray(target) && Array.isArray(source)) {
         source.forEach(function (item, i) {
             if (has.call(target, i)) {
-                if (target[i] && typeof target[i] === 'object') {
+                if (target[i] && typeof target[i] === 'object' && item && typeof item === 'object') {
```

Comment:
please revert this; the dist file should only be updated at release time.
```suggestion
                if (target[i] && typeof target[i] === 'object') {
```